modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsControllerEpicsCommon.scala

```diff
@@ -341,13 +338,8 @@ object TcsControllerEpicsCommon {
     override def setNodChopProbeTrackingConfig(s: TcsEpics.ProbeGuideCmd[F])(c: NodChopTrackingConfig): F[Unit] =
       s.setNodachopa(encode(c.get(NodChop(Beam.A, Beam.A)))) *>
         s.setNodachopb(encode(c.get(NodChop(Beam.A, Beam.B)))) *>
-        s.setNodachopc(encode(c.get(NodChop(Beam.A, Beam.C)))) *>
```

Comment:
I remove most references to beam C because it is not used, and that simplifies the code.